releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,11 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.1.0-preview - October 24th, 2024
+### Enhancements
+* Adds additional telemetry metadata field to feature flags
+  * `AllocationId` is a hash that represents the state of allocation on the flag. When the hash changes, users may receieve a different variant or variant value than before.
```

Comment:
Do we expect/want people to use `AllocationId`? I don't think we have to explain what it does here unless it has a use. You can probably remove the last sentence and just make it all one bullet point

Reply:
Thats a good question. I'm not sure. I feel the need to explain it here since we're not planning to add it to the docs yet. Although.

Reply:
Going to merge for now but can rip it out if we decide to.

Reply:
How do we like this? We don't need to talk about whether it's a hash or something else.

`AllocationId` represents the version of an allocation on a feature flag. The AllocationId changes when there are essential modifications to the allocation, such as adjustments to the percentile or variant configuration, but not for non-essential changes like feature flag descriptions.